lutris/util/system.py

```diff
@@ -45,6 +45,10 @@
 
 IS_64BIT = sys.maxsize > 2**32
 
+NO_TRAY_SUPPORT = [
+    'gnome',
```

Comment:
Perhaps, but what about those that don't use that extension? Is there a way to detect the presence of such an extension?

Reply:
You can search `/home/alex/.local/share/gnome-shell/extensions` for a folder of an an extension that contains `TopIcons` in its name. But there are several versions of those so don't rely on detecting just one kind.

Reply:
What happens if you just instantiate that tray icon no matter what? I doubt that it will bring down the world in flames. Usually, programs with a tray icon don't care if the DE supports them or not, because they shouldn't, it's not the program's responsibility to take care of that.

Reply:
Discord does that. But it has a toggle to disable tray icon for special DEs like Gnome.